sssom/util.py

```diff
@@ -219,3 +226,153 @@ def sha256sum(filename):
         for n in iter(lambda: f.readinto(mv), 0):
             h.update(mv[:n])
     return h.hexdigest()
+
+def merge_msdf(msdf1:MappingSetDataFrame, msdf2:MappingSetDataFrame, reconcile:bool=True, inplace:bool=False) -> MappingSetDataFrame:
```

Comment:
didnt we have this being a method in the msdf rather than as a utility method? similar to pd.merge()?

Reply:
I started it as being part of Class `msdf` but when I had to import `filter_redundant_rows`, it threw an error (I think something about importing x before y and hence may behave unexpectedly .. ) and not doing anything. So I moved the methods where `filter_redundant_rows` was located. We could always put it somewhere else.